app/components/user-dropdown.tsx

```diff
@@ -25,13 +25,15 @@ export function UserDropdown() {
 						// this is for progressive enhancement
 						onClick={(e) => e.preventDefault()}
 						className="flex items-center gap-2"
+						aria-label="User menu"
 					>
 						<Img
 							className="size-8 rounded-full object-cover"
 							alt={user.name ?? user.username}
 							src={getUserImgSrc(user.image?.objectKey)}
 							width={256}
 							height={256}
+							aria-hidden="true"
```

Comment:
The user avatar is a decorative element in this context and should be disregarded accessibility-wise. Even its `alt` is not needed, but I fear to anger eslint at this point.